src/Catty/Extension&Delegate&Protocol/Extensions/Notification/NotificationExtension.swift

```diff
@@ -49,4 +50,5 @@ public class NotificationName: NSObject {
     public static let projectInvalidXml = "Project.invalidXml"
     public static let projectFetchDetailsFailure = "Project.fetchDetailsFailure"
     public static let settingsCrashReportingChanged = "SettingsTableViewController.crashReportingChanged"
+    public static let projectSearchFailure = "Project.searchFailure"
```

Comment:
Minor suggestion: please cluster all project-related notifications (e.g. move it above `settingsCrashReportingChanged`).